### PR TITLE
Jira Ticket https://jira.oicr.on.ca/browse/GR-780

### PR DESCRIPTION
**Pull request by ijafri26**
Update to Dashi 1.0

### CODE REVIEW

**Comment by slazicoicr** [comment]
Do all the other visualizers work with Dash 1.0? We only want to merge into master once all of them work under Dash 1.0.

File: poolqc/pooling_qc_sample.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
 import dash.dependencies as dep
 import dash_table as dt
 import pandas
-import plotly.plotly as plotly
+import chart_studio.plotly as plotly
```
Comment (slazicoicr) [inline poolqc/pooling_qc_sample.py]: This module is never used (neither is `import dash_table.FormatTemplate as FormatTemplate`)

**Comment by callunity** [comment]
I'm seeing a project histogram error:
```
File "/home/harmstrong/workspace/dashi/runreport/proj_hist.py", line 116, in create_coverage_dist
    ['Selected Run', 'All Other Runs'],
  File "/home/harmstrong/workspace/dashi/venv/lib/python3.6/site-packages/plotly/figure_factory/_distplot.py", line 172, in create_distplot
    validate_distplot(hist_data, curve_type)
  File "/home/harmstrong/workspace/dashi/venv/lib/python3.6/site-packages/plotly/figure_factory/_distplot.py", line 46, in validate_distplot
    raise ImportError("FigureFactory.create_distplot requires scipy")
ImportError: FigureFactory.create_distplot requires scipy
```

**Comment by slazicoicr** [comment]
This is fixed by installing scipy `pip install scipy`. I am not sure why plotly does not install it as one of its requirements. 

To avoid this, we should create a setup.py file for this project. I have created a ticket for this https://jira.oicr.on.ca/browse/GR-782

**Pull request merged**